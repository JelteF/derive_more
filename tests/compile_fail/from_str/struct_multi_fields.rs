#[derive(derive_more::FromStr)]
pub struct Foo {
    foo: i32,
    bar: i32,
}

fn main() {}
