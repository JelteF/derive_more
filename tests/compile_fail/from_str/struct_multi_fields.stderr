error: only structs with single field can derive `FromStr`
 --> tests/compile_fail/from_str/struct_multi_fields.rs:2:16
  |
2 |   pub struct Foo {
  |  ________________^
3 | |     foo: i32,
4 | |     bar: i32,
5 | | }
  | |_^
